### PR TITLE
Fish factory import room structures

### PR DESCRIPTION
**Pull request by mefriis**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Introduces room structures and lighting to the bleeding room scene in FishFactory. 

* **What is the current behavior?** (You can also link to an open issue here)
#317 
#250 

* **What is the new behavior?** (if this is a feature change)

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
None

* **Other information**:

### CODE REVIEW

File: Assets/Boats.meta
[the remark this answers is not in the document]
Comment (Herman1412) [reply Assets/Boats.meta:1]: Kan hende vi skal legge de med, kan høre på fredag

File: .DS_Store
[the remark this answers is not in the document]
Comment (Herman1412) [reply .DS_Store:1]: Tror ikke de skal være med, ser noen av de havner i foldere utenfor fish factory

[the remark this answers is not in the document]
Comment (jonp-h) [reply .DS_Store:1]: @mefriis fjern DS_store filene

**Review by jonp-h — APPROVED** [review]

**Review by Herman1412 — COMMENTED** [review]
The extra rooms that are closed off not in use are not necessary as we intend to have a separate scene for each task. 
Fish tank also has a void material that needs to be changed.
Apart from a door and windows it looks good. Door can be added later when we need it.

**Review by jonp-h — APPROVED** [review]

**Pull request closed without merging**